setup.py

```diff
@@ -11,6 +11,7 @@
     keywords='hdf5 h5py',
     url='https://github.com/INM-6/h5py_wrapper',
     packages=['h5py_wrapper', 'tests'],
+    python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*',
```

Comment:
Do I understand it correctly, that this means "not compatible with 3.0" etc? Why is that again? Or does this just mean its not tested with these?

Reply:
I can't quite remember why that is, but I think it is not compatible with these versions, yes.